wsdl/sandbox/2015_2.wsdl

```diff
@@ -2623,7 +2623,7 @@
 	</binding>
 	<service name="NetSuiteService">
 		<port name="NetSuitePort" binding="tns:NetSuiteBinding">
-			<soap:address location="https://webservices.sandbox.netsuite.com/services/NetSuitePort_2015_2"/>
+			<soap:address location="https://webservices.netsuite.com/services/NetSuitePort_2015_2"/>
 		</port>
 	</service>
 </definitions>
```

Comment:
Why do we even need this file anymore if we are using the same wsdl for production and staging?

Reply:
Didn't want to change *too* much at once, but am totally down to consolidate it all.

Is there a reason we're caching them at all?